Multi-Chain/osmosis-cosmos-bridge/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "osmosis-cosmos-ibc",
+  "version": "0.0.1",
+  "description": "This project can be use as a starting point for developing your Cosmos (Cosmos Hub) based SubQuery project",
```

Update to match the readme
```suggestion
  "description": "This SubQuery project indexes all IBC transfer events between Cosmos Hub and Osmosis",
```